bundles/binding/org.openhab.binding.km200/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="km200"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>km200 Binding</name>
+    <description>The KM200 Binding is communicating with a Buderus Logamatic web KM200 / KM100 / KM50. It is possible to recive and send parameters like string or float values.</description>
+    <author>Markus Eckhardt</author>
+    
+    <service-id>org.openhab.binding.km200</service-id>
+
+    <config-description>
+        <parameter name="ip4_address" type="text">
+            <label>IP4 Address</label>
+            <description>IP4 Address of the KMXXX device</description>
+            <default>"192.168.XXX.XXX"</default>
```

Comment:
Are you sure a quoted string with X's in it is a valid default here?  Best to not have a default unless it can be concretely correct.  Please see [this link](https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-configuration-descriptions) for the full XML definition for `<config-description>` and browse other 1.x add-on source for other examples.